BundleTransport/app/src/main/java/net/discdd/bundletransport/UsbFragment.java

```diff
@@ -211,7 +239,6 @@ private void checkUsbConnection(int tries) {
         usbConnected = !usbManager.getDeviceList().isEmpty();
         getActivity().getMainExecutor().execute(() -> {
             if (!usbManager.getDeviceList().isEmpty()) {
-                updateUsbStatus(true, getString(R.string.usb_connection_detected), Color.GREEN);
```

Comment:
did you mean to delete this? if so you probably just want the if (usbManager.getDeviceList().isEmpty()) condition.